lldb/include/lldb/Utility/FileSpec.h

```diff
@@ -443,6 +455,9 @@ class FileSpec {
   /// The unique'd filename path.
   ConstString m_filename;
 
+  /// The optional MD5 checksum of the file.
+  Checksum m_checksum;
+
```

Comment:
This will increase the byte size of every FileSpec object by 16 bytes which is not good. Can we store the checksum somewhere else? Or make a lldb_private::ChecksumFile which has a lldb_private::FileSpec + Checksum?
```
(lldb) p sizeof(llvm::MD5::MD5Result)
(unsigned long) 16
(lldb) p sizeof(lldb_private::FileSpec)
(unsigned long) 24
```

Reply:
The DWARF 5 line table should have a file list where can we easily compute this and warn if needed. It would be fine to store an extra bit in lldb_private::FileSpec that specifies if the checksum has been verified or not, but we would like to keep lldb_private::FileSpec to be 24 bytes or less.